solutions/3772/01.py

```
@@ -1,35 +1,50 @@
 class Solution:
-    def maxSubgraphScore(self, n: int, edges: List[List[int]], good: List[int]) -> List[int]:
+    def maxSubgraphScore(
+        self, n: int, edges: List[List[int]], good: List[int]
```

Ruff (0.14.8)

Undefined name `List`

(F821)